app/src/androidTest/java/org/mozilla/fenix/ui/PerformanceStartupTest.kt

```diff
@@ -28,6 +30,19 @@ private const val FAILURE_MSG = """StrictMode startup suppression count does not
 
 """
 
+private const val RUNBLOCKING_FAILURE_MSG = """RunblockingCounter startup count does not match expected count.
+    
+    If this PR removed code that removed a RunBlocking call, great! Please decrement the suppression count.
+    
+    Did this PR add or call code that incremented the RunBlockingCounter?
+    Did you know that using runBlocking can have negative perfomance implications?
+
+    If so, please do your best to implement a solution without blocking the main thread.
+    Please consult the perf team if you have questions or believe that using runBlocking
+    is the optimal solution.
+
+"""
+
 /**
  * A performance test to limit the number of StrictMode suppressions on startup.
  * This test was written by the perf team.
```

Comment:
We changed the file name: we need to change the class name and test name too.

I think `PerformanceStartupTest` is too general: how about `StartupExcessiveResourceUseTest` or similar?

Reply:
I think that could work, I just changed it to that. I have some qualms with `StartupExcessiveResourceUseTest` since it specifies that we're checking for resources which to me sounds like we're checking for memory and other type of hardware related resources. However, I can't think of a better name that isn't longer or more concise with similar number of letters.